packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -143,30 +137,32 @@ const DarkBackgroundMenu = styled.div`
 // eslint-disable-next-line complexity, react/display-name
 export const Webchat = forwardRef((props, ref) => {
   const {
-    webchatState,
     addMessage,
     addMessageComponent,
-    updateMessage,
-    updateReplies,
-    updateLatestInput,
-    updateTyping,
-    updateWebview,
-    updateSession,
-    updateLastRoutePath,
-    updateHandoff,
-    updateTheme,
-    updateDevSettings,
-    toggleWebchat,
-    toggleEmojiPicker,
-    togglePersistentMenu,
-    toggleCoverComponent,
+    clearMessages,
     doRenderCustomComponent,
+    openWebviewT,
```

Comment:
Doubt: The `T` in this `openWebviewT` is a typo? Or does it mean something?

Reply:
I don't really know, it looks like I have touched this line but I have only sorted alphabetically.